### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by raphaelshirley**
Added basic installation instructions. These worked on mac in a fresh conda environment.

## Code Quality
- [ ] My code follows [the code style of this project](https://rail-hub.readthedocs.io/en/latest/source/contributing.html#naming-conventions)
- [ ] I have written unit tests or justified all instances of `#pragma: no cover`; in the case of a bugfix, a new test that breaks as a result of the bug has been added
- [ ] My code contains relevant comments and necessary documentation for future maintainers; the change is reflected in applicable demos/tutorials (with output cleared!) and added/updated docstrings use the [NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html)
- [ ] Any breaking changes, described above, are accompanied by backwards compatibility and deprecation warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,18 @@ template can not do these for you, otherwise it would :)
...
+in [RAIL](https://github.com/LSSTDESC/RAIL). This package in particular
+is concerned with wrapping the [LePHARE](https://gitlab.lam.fr/Galaxies/LEPHARE/) template fitting code within RAIL.
+
+### Installation
+
+This package can be installed from source using the following commands:
+```console
+git clone https://github.com/LSSTDESC/rail_lephare.git
+cd rail_lephare
+conda install -c conda-forge pytables sqlite
```
Comment (drewoldag) [inline README.md:44]: I just took a swing at this on my work computer (M1 mac) and I only needed to conda install pytables. I didn't have to include sqlite in the conda install. 

I don't think it hurts anything to include it, but I'm not sure it's necessary.

**Review by drewoldag — APPROVED** [review]
This looks good to me, I left one comment regarding conda installing sqlite. All in all though, it looks good.

**Comment by raphaelshirley** [comment]
Yes, you are right. I retried without sqlite and had no problems. I added it to solve an error I was getting with pytest but realised that was also due to the missing pytables originally. I have removed the sqlite install.

Longer term is it possible to install pytables automatically using the pyproject.toml requirements or can that not access conda-forge?

**Comment by drewoldag** [comment]
It's a little tricky simply because some of the dependencies are only in PyPI and others are available on both conda-forge and PyPI. 
 
I believe that this is a rough edge that _RAIL_ has, and it's not isolated to rail_lephare. So, theoretically anyone who has RAIL installed and working should be more of less familiar with this annoyance.

**Review by drewoldag — APPROVED** [review]

**Pull request merged**